text/0039-dust-protection-based-on-byte-costs/0039-dust-protection-based-on-byte-costs.md

```diff
@@ -0,0 +1,1305 @@
+                <td>Milestone Index</td>
+                <td><code>data</code></td>
+                <td>4</td>
+                <td>4</td>
+                <td>-</td>
+            </tr>
+        </table>
+      </details>
+      <details>
+        <summary>Expiration Unix Block</summary>
+        <table>
+            <tr>
+                <td><b>Field</b></td>
+                <td><b>Field type</b></td>
+                <td><b>Length Minimum</b></td>
+                <td><b>Length Maximum</b></td>
+                <td><b>Remarks</b></td>
+            </tr>
+            <tr>
+                <td>Block Type</td>
+                <td><code>data</code></td>
+                <td>1</td>
+                <td>1</td>
+                <td>-</td>
+            </tr>
+            <tr>
+                <td>Unix Time</td>
+                <td><code>data</code></td>
+                <td>4</td>
+                <td>4</td>
+                <td>-</td>
+            </tr>
+        </table>
+      </details>        
+      <details>
+        <summary>Indexation Block</summary>
+        <table>
+            <tr>
+                <td><b>Field</b></td>
+                <td><b>Field type</b></td>
+                <td><b>Length Minimum</b></td>
+                <td><b>Length Maximum</b></td>
+                <td><b>Remarks</b></td>
+            </tr>
+            <tr>
+                <td>Block Type</td>
+                <td><code>data</code></td>
+                <td>1</td>
+                <td>1</td>
+                <td>-</td>
+            </tr>
+            <tr>
+                <td>Indexation Data Length</td>
+                <td><code>data</code></td>
+                <td>1</td>
+                <td>1</td>
+                <td>-</td>
+            </tr>
+            <tr>
+                <td>Indexation Data</td>
+                <td><code>key+data</code></td>
+                <td>64</td>
+                <td>64</td>
+                <td>If used in combination with the sender block, the weight  of this field doubles.</td>
+            </tr>            
+        </table>
+      </details>        
+      <details>
+        <summary>Metadata Block</summary>
+        <table>
+            <tr>
+                <td><b>Field</b></td>
+                <td><b>Field type</b></td>
+                <td><b>Length Minimum</b></td>
+                <td><b>Length Maximum</b></td>
+                <td><b>Remarks</b></td>
+            </tr>
+            <tr>
+                <td>Block Type</td>
+                <td><code>data</code></td>
+                <td>1</td>
+                <td>1</td>
+                <td>-</td>
+            </tr>
+            <tr>
+                <td>Metadata Data Length</td>
+                <td><code>data</code></td>
+                <td>4</td>
+                <td>4</td>
+                <td>-</td>
+            </tr>
+            <tr>
+                <td>Metadata Data</td>
+                <td><code>data</code></td>
+                <td>0</td>
```

Comment:
According to https://github.com/lzpap/protocol-rfcs/blob/17bf6d4aaabff7f667df21a3b3137782e74c481c/text/0038-output-types-for-tokenization-and-sc/0038-output-types-for-tokenization-and-sc.md?plain=1#L1301
the minimum should be 1, the same in lines 700, 871 and 1242